nautobot/extras/tests/test_customfields.py

```diff
@@ -334,6 +399,9 @@ def test_get_single_object_without_custom_field_data(self):
 
         response = self.client.get(url, **self.header)
         self.assertEqual(response.data["name"], self.sites[0].name)
+        # A model directly instantiated via the ORM does NOT automatically receive custom field default values.
+        # This is arguably a bug.
```

Comment:
Not sure how we would solve this since it would be auto-magically pulling in related fields on instantiation which is likely not what we want due to possible side effects. All the same, I argue that is at least violating the principal of least astonishment and we should have an issue to track it for v2.